content/search-github/getting-started-with-searching-on-github/understanding-the-search-syntax.md

```diff
@@ -35,7 +35,7 @@ Query  | Example
 
 ## Query for values between a range
 
-You can use the range syntax <code><em>n</em>..<em>n</em></code> to search for values within a range, where the first number _n_ is the lowest value and the second is the highest value.
+You can use the range syntax <code><em>n</em>..<em>n</em></code> to search for values within a range, where the first number NUMBER is the lowest value and the second is the highest value.
```

Comment:
We either need to update the placeholders in the following tables to be "NUMBER", or use "N".

```suggestion
You can use the range syntax <code><em>n</em>..<em>n</em></code> to search for values within a range, where the first number N is the lowest value and the second is the highest value.
```